src/plugins/slowed/index.tsx

```
@@ -0,0 +1,140 @@
+  name: () => 'Slowed',
+  authors: ['The-Kryz'],
+  restartNeeded: false,
+  config: DEFAULT_CONFIG,
+  stylesheets: [style],
+
+  renderer: {
+    cleanup: null as (() => void) | null,
+
+    start({ config, setConfig }) {
+      const safeConfig = config || DEFAULT_CONFIG;
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)